src/vstart.sh

```diff
@@ -255,7 +256,8 @@ options:
 	--no-restart: dont restart process when using ceph-run
 	--jaeger: use jaegertracing for tracing
 	--seastore-devs: comma-separated list of blockdevs to use for seastore
-	--seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore
+	--seastore-secondary-devs: comma-separated list of secondary blockdevs to use for seastore
+	--seastore-secondary-devs-type: device type of all secondary blockdevs, SSD or HDD
```

Comment:
Shouldn't this list match `DD, SSD(default), ZNS and RANDOM_BLOCK_SSD.` on line 199 of `crimson.rst`?

Reply:
Yeah, you are right, they should be consistent.